src/renderer/components/+cluster-settings/cluster-settings.tsx

```diff
@@ -16,6 +16,7 @@ import { getMatchedCluster } from "../cluster-manager/cluster-view.route";
 export class ClusterSettings extends React.Component {
   render() {
     const cluster = getMatchedCluster();
+    if (!cluster) return null;
```

Comment:
Did you catch some error or just added to be "more safe"?

Reply:
It prevents error when you try to disconnect cluster while located in it's `Cluster Settings`.